### DISCUSSION TITLE
BUG: Underscores in embedded LaTeX in Markdown

### DISCUSSION THREAD

**Discussion by kylechui** [community/community#58722] [Repositories]
### Select Topic Area

Bug

### Body

The following LaTeX code yields the error `'_' allowed only in math mode`:
```latex
$\texttt{INT\_LITERAL}$
```
I tried escaping the backslash, i.e. making it `INT\\_LITERAL`, and then it rendered properly. I suspect there is an extra parsing step that is processing the backslashes and reducing `\_` &rarr; `_` before passing it into the LaTeX engine.

While using `\\_` is a viable workaround for the time being, it is incompatible with my local renderer and forces me to choose if I want my files to be previewable locally, or remotely (in my repo's README). Any guidance would be much appreciated!

If there is a "more appropriate" place to report this, please let me know.

**Comment by nschloe** [comment]
I'd never use the $-syntax; it's super buggy. Best go with the backticks like

```markdown
$`\texttt{INT\_LITERAL}`$
```

> $`\texttt{INT\_LITERAL}`$

**Comment by cosing** [comment]
`word $\text{SampleRT}_\text{t4}$ 、$\text{SampleRT}_\text{t3}$ word`
It should be (I add extra \ before \_)
word $\text{SampleRT}\_\text{t4}$ and $\text{SampleRT}\_\text{t3}$ word
not 
word $\text{SampleRT}_\text{t4}$ and $\text{SampleRT}_\text{t3}$ word

**Reply by cosing** [reply]
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

> There are two options for delimiting a math expression inline with your text. You can either surround the expression with dollar symbols ($), or start the expression with <code>$`</code> 

> and end it with <code>`$</code>. The latter syntax is useful when the expression you are writing contains characters that overlap with markdown syntax. For more information, see [Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).
